************* Module reusable_lib.reusable_db
reusable_db.py:42:0: W1405: Quote delimiter " is inconsistent with the rest of the file (inconsistent-quotes)
reusable_db.py:55:0: W1405: Quote delimiter " is inconsistent with the rest of the file (inconsistent-quotes)
reusable_db.py:14:11: W0621: Redefining name 'db' from outer scope (line 58) (redefined-outer-name)
reusable_db.py:14:15: W0621: Redefining name 'h' from outer scope (line 57) (redefined-outer-name)
reusable_db.py:21:4: W0621: Redefining name 'output' from outer scope (line 59) (redefined-outer-name)
reusable_db.py:22:8: C0103: Variable name "_h" doesn't conform to '^[a-z][a-z0-9_]*$' pattern (invalid-name)
reusable_db.py:34:12: W0621: Redefining name 'db' from outer scope (line 58) (redefined-outer-name)
reusable_db.py:34:16: W0621: Redefining name 'k' from outer scope (line 56) (redefined-outer-name)
reusable_db.py:34:19: W0621: Redefining name 'h' from outer scope (line 57) (redefined-outer-name)
reusable_db.py:34:22: W0621: Redefining name 'output' from outer scope (line 59) (redefined-outer-name)
reusable_db.py:40:8: W0621: Redefining name 'i' from outer scope (line 60) (redefined-outer-name)
reusable_db.py:34:16: W0613: Unused argument 'k' (unused-argument)
reusable_db.py:10:0: W0611: Unused import reusable_vt (unused-import)
************* Module reusable_lib.reusable_json
reusable_json.py:1:0: C0114: Missing module docstring (missing-module-docstring)
reusable_json.py:8:11: W0703: Catching too general exception Exception (broad-except)
reusable_json.py:6:13: W1514: Using open without explicitly specifying an encoding (unspecified-encoding)
reusable_json.py:20:11: W0703: Catching too general exception Exception (broad-except)
reusable_json.py:18:13: W1514: Using open without explicitly specifying an encoding (unspecified-encoding)
************* Module reusable_lib.reusable_sql
reusable_sql.py:73:0: C0301: Line too long (386/80) (line-too-long)
reusable_sql.py:74:0: C0301: Line too long (86/80) (line-too-long)
reusable_sql.py:25:0: W1405: Quote delimiter " is inconsistent with the rest of the file (inconsistent-quotes)
reusable_sql.py:49:0: W1405: Quote delimiter " is inconsistent with the rest of the file (inconsistent-quotes)
reusable_sql.py:86:0: W1405: Quote delimiter " is inconsistent with the rest of the file (inconsistent-quotes)
reusable_sql.py:1:0: C0114: Missing module docstring (missing-module-docstring)
reusable_sql.py:88:4: W0105: String statement has no effect (pointless-string-statement)
************* Module reusable_lib.reusable_hash
reusable_hash.py:30:0: W1405: Quote delimiter " is inconsistent with the rest of the file (inconsistent-quotes)
reusable_hash.py:93:0: W1405: Quote delimiter " is inconsistent with the rest of the file (inconsistent-quotes)
reusable_hash.py:1:0: C0114: Missing module docstring (missing-module-docstring)
reusable_hash.py:13:23: W0622: Redefining built-in 'format' (redefined-builtin)
reusable_hash.py:19:4: W0105: String statement has no effect (pointless-string-statement)
reusable_hash.py:25:15: W0622: Redefining built-in 'format' (redefined-builtin)
reusable_hash.py:37:4: W0105: String statement has no effect (pointless-string-statement)
reusable_hash.py:51:8: C0103: Variable name "_v" doesn't conform to '^[a-z][a-z0-9_]*$' pattern (invalid-name)
reusable_hash.py:3:0: W0611: Unused import pprint (unused-import)
************* Module reusable_lib.reusable_strings
reusable_strings.py:8:0: W1405: Quote delimiter ' is inconsistent with the rest of the file (inconsistent-quotes)
reusable_strings.py:15:0: W1405: Quote delimiter ' is inconsistent with the rest of the file (inconsistent-quotes)
reusable_strings.py:1:0: C0114: Missing module docstring (missing-module-docstring)
reusable_strings.py:7:11: W0703: Catching too general exception Exception (broad-except)
************* Module reusable_lib.reusable_ldd
reusable_ldd.py:23:0: W1405: Quote delimiter " is inconsistent with the rest of the file (inconsistent-quotes)
reusable_ldd.py:1:0: C0114: Missing module docstring (missing-module-docstring)
reusable_ldd.py:13:12: W0621: Redefining name 'd' from outer scope (line 24) (redefined-outer-name)
reusable_ldd.py:18:11: W0703: Catching too general exception Exception (broad-except)
reusable_ldd.py:12:8: C0103: Variable name "_deps" doesn't conform to '^[a-z][a-z0-9_]*$' pattern (invalid-name)
************* Module reusable_lib.template
template.py:1:0: C0114: Missing module docstring (missing-module-docstring)
************* Module reusable_lib.reusable_vt
reusable_vt.py:62:0: C0301: Line too long (128/80) (line-too-long)
reusable_vt.py:1:0: C0114: Missing module docstring (missing-module-docstring)
reusable_vt.py:7:13: W0621: Redefining name 'k' from outer scope (line 59) (redefined-outer-name)
reusable_vt.py:7:16: W0621: Redefining name 'h' from outer scope (line 60) (redefined-outer-name)
reusable_vt.py:8:4: W0621: Redefining name 'output' from outer scope (line 64) (redefined-outer-name)
reusable_vt.py:10:8: W0105: String statement has no effect (pointless-string-statement)
reusable_vt.py:24:15: W0621: Redefining name 'db' from outer scope (line 63) (redefined-outer-name)
reusable_vt.py:24:19: W0621: Redefining name 'k' from outer scope (line 59) (redefined-outer-name)
reusable_vt.py:24:22: W0621: Redefining name 'h' from outer scope (line 60) (redefined-outer-name)
reusable_vt.py:25:4: W0621: Redefining name 'output' from outer scope (line 64) (redefined-outer-name)
************* Module reusable_lib.reusable_utils
reusable_utils.py:38:0: C0301: Line too long (86/80) (line-too-long)
reusable_utils.py:17:0: W1405: Quote delimiter " is inconsistent with the rest of the file (inconsistent-quotes)
reusable_utils.py:38:0: W1405: Quote delimiter " is inconsistent with the rest of the file (inconsistent-quotes)
reusable_utils.py:42:0: W1405: Quote delimiter " is inconsistent with the rest of the file (inconsistent-quotes)
reusable_utils.py:57:0: W1405: Quote delimiter " is inconsistent with the rest of the file (inconsistent-quotes)
reusable_utils.py:1:0: C0114: Missing module docstring (missing-module-docstring)
reusable_utils.py:23:4: W0702: No exception type(s) specified (bare-except)
reusable_utils.py:37:4: C0103: Variable name "logFormatter" doesn't conform to '^[a-z][a-z0-9_]*$' pattern (invalid-name)
reusable_utils.py:40:4: C0103: Variable name "rootLogger" doesn't conform to '^[a-z][a-z0-9_]*$' pattern (invalid-name)
reusable_utils.py:42:4: C0103: Variable name "fileHandler" doesn't conform to '^[a-z][a-z0-9_]*$' pattern (invalid-name)
reusable_utils.py:42:38: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)
reusable_utils.py:46:4: C0103: Variable name "consoleHandler" doesn't conform to '^[a-z][a-z0-9_]*$' pattern (invalid-name)
************* Module reusable_lib.reusable_stat
reusable_stat.py:70:0: C0301: Line too long (83/80) (line-too-long)
reusable_stat.py:86:0: C0301: Line too long (85/80) (line-too-long)
reusable_stat.py:122:0: C0301: Line too long (194/80) (line-too-long)
reusable_stat.py:129:0: C0301: Line too long (85/80) (line-too-long)
reusable_stat.py:328:0: C0301: Line too long (118/80) (line-too-long)
reusable_stat.py:18:0: W1405: Quote delimiter " is inconsistent with the rest of the file (inconsistent-quotes)
reusable_stat.py:28:0: W1405: Quote delimiter " is inconsistent with the rest of the file (inconsistent-quotes)
reusable_stat.py:30:0: W1405: Quote delimiter " is inconsistent with the rest of the file (inconsistent-quotes)
reusable_stat.py:30:0: W1405: Quote delimiter " is inconsistent with the rest of the file (inconsistent-quotes)
reusable_stat.py:231:0: W1405: Quote delimiter " is inconsistent with the rest of the file (inconsistent-quotes)
reusable_stat.py:242:0: W1405: Quote delimiter " is inconsistent with the rest of the file (inconsistent-quotes)
reusable_stat.py:339:0: W1405: Quote delimiter " is inconsistent with the rest of the file (inconsistent-quotes)
reusable_stat.py:340:0: W1405: Quote delimiter " is inconsistent with the rest of the file (inconsistent-quotes)
reusable_stat.py:341:0: W1405: Quote delimiter " is inconsistent with the rest of the file (inconsistent-quotes)
reusable_stat.py:342:0: W1405: Quote delimiter " is inconsistent with the rest of the file (inconsistent-quotes)
reusable_stat.py:343:0: W1405: Quote delimiter " is inconsistent with the rest of the file (inconsistent-quotes)
reusable_stat.py:344:0: W1405: Quote delimiter " is inconsistent with the rest of the file (inconsistent-quotes)
reusable_stat.py:345:0: W1405: Quote delimiter " is inconsistent with the rest of the file (inconsistent-quotes)
reusable_stat.py:350:0: W1405: Quote delimiter " is inconsistent with the rest of the file (inconsistent-quotes)
reusable_stat.py:1:0: C0114: Missing module docstring (missing-module-docstring)
reusable_stat.py:33:52: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)
reusable_stat.py:50:11: W0703: Catching too general exception Exception (broad-except)
reusable_stat.py:56:0: C0103: Function name "getSystemInfo" doesn't conform to '^(?:(?P<exempt>setUp|tearDown|setUpModule|tearDownModule)|(?P<camel_case>_?[A-Z][a-zA-Z0-9]*)|(?P<snake_case>_?[a-z][a-z0-9_]*))$' pattern (invalid-name)
reusable_stat.py:113:11: W0703: Catching too general exception Exception (broad-except)
reusable_stat.py:118:0: C0103: Function name "parse_W_user" doesn't conform to '^(?:(?P<exempt>setUp|tearDown|setUpModule|tearDownModule)|(?P<camel_case>_?[A-Z][a-zA-Z0-9]*)|(?P<snake_case>_?[a-z][a-z0-9_]*))$' pattern (invalid-name)
reusable_stat.py:153:8: E0602: Undefined variable 'log' (undefined-variable)
reusable_stat.py:153:18: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)
reusable_stat.py:203:23: W0703: Catching too general exception Exception (broad-except)
reusable_stat.py:205:4: W0105: String statement has no effect (pointless-string-statement)
reusable_stat.py:232:11: W0703: Catching too general exception Exception (broad-except)
reusable_stat.py:244:11: W0703: Catching too general exception Exception (broad-except)
reusable_stat.py:252:0: C0103: Function name "getW" doesn't conform to '^(?:(?P<exempt>setUp|tearDown|setUpModule|tearDownModule)|(?P<camel_case>_?[A-Z][a-zA-Z0-9]*)|(?P<snake_case>_?[a-z][a-z0-9_]*))$' pattern (invalid-name)
reusable_stat.py:258:11: W0703: Catching too general exception Exception (broad-except)
reusable_stat.py:256:12: W1510: Using subprocess.run without explicitly set `check` is not recommended. (subprocess-run-check)
reusable_stat.py:286:0: W0105: String statement has no effect (pointless-string-statement)
reusable_stat.py:307:15: W0703: Catching too general exception Exception (broad-except)
reusable_stat.py:311:15: W0703: Catching too general exception Exception (broad-except)
reusable_stat.py:315:15: W0703: Catching too general exception Exception (broad-except)
reusable_stat.py:321:15: W0703: Catching too general exception Exception (broad-except)
reusable_stat.py:325:15: W0703: Catching too general exception Exception (broad-except)
reusable_stat.py:330:15: W0703: Catching too general exception Exception (broad-except)
reusable_stat.py:307:8: W0612: Unused variable 'ex' (unused-variable)
reusable_stat.py:1:0: W0611: Unused import sys (unused-import)

-----------------------------------
Your code has been rated at 7.63/10

